### PR TITLE
[14.0][MIG] stock_available_unreserved

### PR DESCRIPTION
**Pull request by MateuGForgeFlow**
Standard migration to v14.
cc @LoisRForgeFlow

### CODE REVIEW

File: stock_available_unreserved/models/product.py [hunk 1]
```diff
@@ -34,8 +34,11 @@ def action_open_quants_unreserved(self):
         quant_ids = quants.filtered(
             lambda x: x.product_id.qty_available_not_res > 0
         ).ids
-        result = self.env.ref("stock.product_template_open_quants").read()[0]
-        result["domain"] = [("id", "in", quant_ids)]
+        result = dict()
```
Comment (LoisRForgeFlow) [inline stock_available_unreserved/models/product.py]: This is not correct, if `product_template_open_quants` action does not exist anymore you need to use a new one or complete it here.

**Comment by florian-dacosta** [comment]
@MateuGForgeFlow  Please check https://github.com/OCA/stock-logistics-warehouse/pull/1042

**Comment by LoisRForgeFlow** [comment]
Travis failing because of https://github.com/OCA/stock-logistics-warehouse/issues/1102.

**Comment by LoisRForgeFlow** [comment]
> @MateuGForgeFlow Please check #1042

We will adapt all our modules using stock_available_unreserved to now use stock_free_qty. Thanks for the pointer!

**Comment by MateuGForgeFlow** [comment]
The stock_available_unreserved functionality is covered by #1042. I close the PR. Thank you @florian-dacosta

**Pull request closed without merging**